package.json

```diff
@@ -39,7 +39,7 @@
     "uuid": "^8.3.1"
   },
   "scripts": {
-    "start": "nodemon -w craco.config.js -w ./antd.customize.less --exec \"craco start\"",
+    "start": "nodemon -w craco.config.js -w ./antd.customize.less --exec \"npx craco start\"",
```

Comment:
❓ Will this work for everyone?

Reply:
Yeah, without npx, it crashes and I'm pretty sure npx is default execution command for Node and npm